src/pool/withdraw.ts

```diff
@@ -58,27 +58,26 @@ export function makeWithdrawal(
   const poolDatumMapped = poolArtifacts.poolDatumMapped;
   const poolConfigDatum = poolArtifacts.poolConfigDatum;
 
-  if (balanceToWithdraw <= poolConfigDatum.minTransition) {
+  let valueDelta = amountToWithdraw;
+
+  if (Math.abs(Number(valueDelta)) <= poolConfigDatum.minTransition) {
     throw "Withdraw amount is too small";
   }
 
   // Calculate amount of LP tokens to be minted
```

Comment:
probably should clarify this comment, since it seems like we're actually calculating lps to be burned, not minted.